lane.py

```diff
@@ -548,32 +548,20 @@ def __init__(self,
         # e.g. "All Science Fiction"
         self.include_all_feed = include_all
 
-        def set_from_parent(field_name, value, default=None):
```

Comment:
I moved this method out of `Lane.__init__()` so that other supporting initialization methods could use it.